testing_exercise/testingSpec.js

```diff
@@ -1,3 +1,49 @@
 var expect = chai.expect;
 
-// WRITE YOUR TESTS HERE!
\ No newline at end of file
+// WRITE YOUR TESTS HERE!
+describe("Tests", function(){
+
+	it("replaces lowercase", function(){
+		expect(replaceWith("awesome", "e", "z")).to.be.equal("awzsomz");
+	});
+
+	it("replaces case sensitive", function(){
+		expect(replaceWith("Foo", "F", "B")).to.be.equal("Boo");
+	});
+
+	// unclear test case:
+	// replaceWith("Woo", "W", "p"); // Poo? or poo? thus far unspecified.
```

Comment:
In this case, it should return 'poo' - the casing of the original character doesn't matter, just the casing of the character doing the replacing.